docs/install/openshift.md

```diff
@@ -36,6 +36,45 @@ oc login -u kubeadmin
 By default, the operator is installed into the `argocd-operator-system` namespace. To modify this, update the
 value of the `namespace` specified in the `config/default/kustomization.yaml` file. 
 
+### Conversion Webhook Support
+
+ArgoCD `v1alpha1` CRD has been **deprecated** starting from **argocd-operator v0.8.0**. To facilitate automatic migration of existing `v1alpha1` ArgoCD CRs to `v1beta1`, conversion webhook support has been introduced.
+
+By default, the conversion webhook is disabled for the manual installation of the operator. Users can modify the configurations to enable conversion webhook support using the instructions provided below.
```

Comment:
By "manual" do we mean non-OLM here?

Reply:
Yes. I updated the docs to reflect this.